velox/docs/functions/spark/string.rst

```diff
@@ -89,6 +89,19 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
 
     Returns the length of ``string`` in characters.
 
+.. spark:function:: levenshtein(string, string) -> integer
```

Comment:
nit: maybe use levenshtein(string, string[, threshold]) to illustrate threshold is optional.